mailpoet/lib/WooCommerce/MultichannelMarketing/MPMarketingChannel.php

```diff
@@ -131,4 +158,13 @@ protected function isMPSetupComplete(): bool {
 
     return $version !== null;
   }
+
+  /**
+   * Check if user is sending with the MailPoet sending method
+   * @return bool
+   */
+  protected function isUserSendingWithMSS(): bool {
+    $sendingMethod = $this->settings->get('mta.method', SettingsController::DEFAULT_SENDING_METHOD);
+    return $sendingMethod === Mailer::METHOD_MAILPOET;
+  }
```

Comment:
I would suggest replacing this method with[ `Bridge::isMailpoetSendingServiceEnabled`.](https://github.com/mailpoet/mailpoet/blob/b4b49166f24f2b6412b1374e82ac9ccdc2eb7521/mailpoet/lib/Services/Bridge.php#L79-L87) The logic is not very complex, but I think we should reuse existing code when possible.